### PR TITLE
added content plan to getting started page

### PR DESCRIPTION
**Pull request by jayk-u**
Content plan has been added for the getting started page.
Writing content can be done later. Started to prepare a few screenshots too.

### CODE REVIEW

**Review by paulRbr — APPROVED** [review]
thanks for this new page

File: docs/help/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Deploying Bump.sh
+
+Get started in a few minutes with Bump!
+
+## Create your first documentation
+Step 1: Create your space/documentation
+Step 2: Upload your file
+    - upload file directly
+    - use the CLI
```
Comment (ChristopheDujarric) [inline docs/help/getting-started.md:9]: Shouldn't we add a link to the dedicated doc section anywhere possible?

Comment (jayk-u) [reply docs/help/getting-started.md:9]: Maybe I wasn't clear enough.
This is just the draft of the pin points for our Getting started pages.
The idea is to provide a quick step-by-step of our onboarding. The content isn't ready yet, I'm just throwing here the main ideas to get a first approval.

Every time one of our main features is mentioned in that part, we'll add the link to the dedicated section in our doc. Does it make more sense? WDYT?

Comment (ChristopheDujarric) [reply docs/help/getting-started.md:9]: Thanks for clarifying, looks good to me then :)

**Pull request merged**